packages/editor/src/extensions/code-block/tests/code-block.test.ts

```diff
@@ -191,3 +192,28 @@ test("editing code in a highlighted code block should not be too slow", async ()
   expect(timings.reduce((a, b) => a + b) / timings.length).toBeLessThan(16);
   expect(editorElement.outerHTML).toMatchSnapshot();
 });
+
+test("Switching codeblock language should register the new language", async () => {
+  const editorElement = h("div");
+  const { editor } = createEditor({
+    element: editorElement,
+    initialContent: CODEBLOCKS_HTML,
+    extensions: {
+      codeblock: CodeBlock
+    }
+  });
+
+  const plugin = HighlighterPlugin({
+    name: CodeBlock.name,
+    defaultLanguage: "plaintext"
+  });
+  editor.commands.updateAttributes(CodeBlock.name, { language: "java" });
+  await new Promise((resolve) => setTimeout(resolve, 100));
+
+  const highlighterState = plugin.spec.state?.init(editor.state, editor.state);
+  const highlighterLanguages = highlighterState?.languages;
+  for (const key in highlighterLanguages) {
+    const isTrue = refractor.registered(highlighterLanguages[key]);
+    expect(isTrue).toBe(true);
```

Comment:
This should be a single line or rename the `isTrue` to `isRegistered`.

Reply:
OK